### DISCUSSION TITLE
Page break in template

### DISCUSSION THREAD

**Discussion by YumiMiyoshi** [jxlsteam/jxls#169] [Q&A] [answered]
Hello,
I want to insert a page break in a template.
Is it possible to set page breaks in a template?

↓I tried the following, but I couldn't...
${sheet.setRowBreak(hssfSheet.getRowNum())}

**Comment by YumiMiyoshi** [comment, marked as answer]
I inserted a page break in the created Excel file with Apache POI.

**Reply by Deninc** [reply]
hi @YumiMiyoshi, can you please guide me how did you do it? Can we add the page break dynamically inside the Excel template?

**Reply by YumiMiyoshi** [reply]
@Deninc 
hi, I set a specific character (for example, '☆') in the cell where I want to put a page break in the template, and searched for '☆' in Apache POI → replaced it with a page break.
